CMakeLists.txt

```diff
@@ -848,6 +848,13 @@ endif()
 
 # Add all of the subdirectories, where we actually do work.
 
+# Include stdlib/ here because any targets defining IS_STDLIB
```

Comment:
Can you make this bigger and clearer. My suggestion:

```
*PLEASE READ* We must include the subdirectory stdlib/ here to ensure that stdlib/ is included before tools/. Please do not move this after tools!

<REASONING>
```

And can you put a note on include(tools) as well.